frontend/src/pages/Product/index.js

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import "./checkout.css";
+import product1 from "../../assets/images/item2.jpg";
+import Select from "../../components/Select";
+import Input from "../../components/Input";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faCircleInfo } from "@fortawesome/free-solid-svg-icons";
+
+
+const Checkout = () => {
```

Comment:
Can you rename this to `Product`. Checkout is a different thing and probably will be implemented differently somewhere else